spec/commands/csv_data_upload_spec.rb

```diff
@@ -1,9 +1,10 @@
 require 'rails_helper'
 
 describe 'CsvDataUpload (integration)' do
-  let(:legislations_simple_csv) { fixture_file('legislations_simple.csv') }
-  let(:litigations_simple_csv) { fixture_file('litigations_simple.csv') }
-  let(:companies_simple_csv) { fixture_file('companies_simple.csv') }
+  let(:legislations_csv) { fixture_file('legislations.csv') }
```

Comment:
as @tsubik suggested some time ago, I removed `_simple` suffix as it doesn't bring much value.